examples/_config.yml

```diff
@@ -11,7 +11,7 @@ sphinx:
 
 execute:
   # Exclude some examples from execution (these are still deployed as html pages)
-  exclude_patterns: [".venv/*", "Boxes_QPE_example.ipynb", "Forest_portability_example.ipynb", "backends_example.ipynb", "qiskit_integration.ipynb", "comparing_simulators.ipynb", "expectation_value_example.ipynb", "pytket-qujax_heisenberg_vqe.ipynb", "spam_example.ipynb", "tket_benchmarking.ipynb", "entanglement_swapping.ipynb", "pytket-qujax-classification.ipynb"]
```

Comment:
The commit message "Execute QPE notebook" is misleading here. This was the name of an old file from a previous PR.

The QPE notebook is executed on CI.